config/chainbridge-runtime-config.cere.dev.js

```diff
@@ -38,8 +38,8 @@ window.__RUNTIME_CONFIG__ = {
         type: "Substrate",
         nativeTokenSymbol: "CERE",
         chainbridgePalletName: "chainBridge",
-        bridgeFeeFunctionName: "tokenTransferFee",
-        transferPalletName: "palletBridge",
+        bridgeFeeValue: 0,
```

Comment:
What is the function `tokenTransferFee`? Why did you remove it?

Reply:
We don't have bridge fee implementation in our pallet. So it gives us the error when it tries to fetch the bridge fee via the pallet, to override this issue I have removed the `bridgeFeeFunctionName` and introduced the `bridgeFeeValue` variable.